### PR TITLE
Feature/dashboard

### PR DESCRIPTION
**Pull request by fungjj92**
## Overview

Creates the dashboard page as the app homepage. Two view for if there are or aren't projects. Default set to false. The lab can now be found at [http://localhost:3100/#/lab](http://localhost:3100/#/lab) 

### Demo

screen shot 2016-09-02 at 4 34 51 pm

### Notes

Styling is extremely light. Regular bootstrap is broken...

## Testing Instructions

Change the this.isProject bool to true in dashboard.component.ts to see the alternate view.

### CODE REVIEW

File: src/app/dashboard/dashboard.component.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            <h2>Lab</h2>
+        </div>
+        <div class="dash-right">
+            <button type="button" class="btn btn-primary">New Project</button>
+        </div>
+    </div>
+    <div class="dash-body">
+        <div *ngIf="!isProject">
+            <i class="icon-cloud"></i>
+            <div>To explore climate change data, <a href="#/lab">start a new project.</a></div>
```
Comment (CloudNiner) [inline src/app/dashboard/dashboard.component.html]: `<a>` tags should use the `routerLink` directive

File: src/app/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<div class="app-fullscreen">
+
+    <navbar></navbar>
+
+    <div class="dash-header">
+        <div class="dash-left">
+            <h2>Lab</h2>
+        </div>
+        <div class="dash-right">
+            <button type="button" class="btn btn-primary">New Project</button>
```
Comment (CloudNiner) [inline src/app/dashboard/dashboard.component.html]: This button should probably also trigger a change to the lab page

**Comment by flibbertigibbet** [comment]
Looks like we'll need to pin the bootstrap package version now to avoid pulling in 1.1.1. Could you put that in this PR?

File: src/app/dashboard/dashboard.component.html (hunk 2 above)
Comment (flibbertigibbet) [inline src/app/dashboard/dashboard.component.html]: Looks like we've got some custom classes defined in use elsewhere in the app; this should probably have the classes `button button-default` instead, to match what's on the 'reset' button on the charts page.

File: src/app/dashboard/dashboard.component.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import { NavbarComponent } from '../navbar/navbar.component';
+
+@Component({
+  selector: 'dashboard',
+  templateUrl: './dashboard.component.html',
+  directives: [ NavbarComponent ]
+})
+export class DashboardComponent {
+
+    private isProject: boolean;
```
Comment (flibbertigibbet) [inline src/app/dashboard/dashboard.component.ts]: Maybe rename to `haveProjects`? I'm guessing this is to track whether or not there are any projects to list. Or better yet, define the list of `projects` as an array for which we can check the length.

**Comment by flibbertigibbet** [comment]
+1 from me after addressing above

**Pull request merged**